jnosql-mapping/jnosql-mapping-core/src/main/java/org/eclipse/jnosql/mapping/core/spi/ReflectionEntityMetadataExtension.java

```diff
@@ -0,0 +1,93 @@
+/*
+ *  Copyright (c) 2022 Contributors to the Eclipse Foundation
+ *   All rights reserved. This program and the accompanying materials
+ *   are made available under the terms of the Eclipse Public License v1.0
+ *   and Apache License v2.0 which accompanies this distribution.
+ *   The Eclipse Public License is available at http://www.eclipse.org/legal/epl-v10.html
+ *   and the Apache License v2.0 is available at http://www.opensource.org/licenses/apache2.0.php.
+ *
+ *   You may elect to redistribute this code under either of these licenses.
+ *
+ *   Contributors:
+ *
+ *   Otavio Santana
+ *   Maximillian Arruda
+ */
+package org.eclipse.jnosql.mapping.core.spi;
```

Comment:
Max, this is not core anymore.
It belongs to reflection domain, thus, please, update the package name as well.